src/System.Net.HttpListener/tests/Configurations.props

```diff
@@ -2,7 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
```

Comment:
If we remove the netstandard, and only add the netcoreapp-* for build configurations, then we are not able to run tests for UAP and ILC. 

@davidsh @DavidGoll